### PR TITLE
Add canales s3 functions

### PR DESCRIPTION
**Pull request by sharinetmc**

### CODE REVIEW

**Comment by sharinetmc** [comment]
@shaunagm, just wondering if you would have a chance to take a quick look at this (but also, I can perhaps get someone else, maybe Wily to go through this PR)

**Comment by sharinetmc** [comment]
@willyraedy wondering, if you have the capacity, if you could do a quick pr review of this? Thanks!

**Comment by sharinetmc** [comment]
Hiya @IanRFerguson! Just wondering, if you have the capacity, if you could reveiw these changes but totally understand if you don't have the time!

File: parsons/aws/s3.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import re
 import boto3
 from botocore.client import ClientError
+
+# from parsons.databases.redshift.redshift import Redshift
```
Comment (IanRFerguson) [inline parsons/aws/s3.py:5]: Can we remove this if it's no longer in use?

File: parsons/databases/redshift/redshift.py [hunk 2]
```diff
@@ -811,6 +811,66 @@ def unload(
...
+            escape=escape,
+            allow_overwrite=allow_overwrite,
+            parallel=parallel,
+            max_file_size=max_file_size,
+            aws_region=aws_region,
+        )
+
+        self.query(f"drop table if exists {rs_table} {query_end}")
+
+        return None
```
Comment (IanRFerguson) [inline parsons/databases/redshift/redshift.py:872]: You don't technically need to `return None` explicitly but it doesn't hurt anything

File: test/test_redshift.py [hunk 3]
```diff
@@ -644,6 +644,28 @@ def test_unload(self):
         # Check that files are there
         self.assertTrue(self.s3.key_exists(self.temp_s3_bucket, "unload_test"))
 
+    def test_drop_and_unload(self):
```
Comment (IanRFerguson) [inline test/test_redshift.py:647]: Need docstring

**Review by IanRFerguson — APPROVED** [review]
Overall looks great! Approved with the caveat that unused imports should be dropped before merging

**Pull request merged**